doc/_static/style.css

```diff
@@ -18,9 +18,9 @@ code {
     color: #2c3e50 !important;
 }
 
-.function dt {
+/* .function dt {
```

Comment:
I think custom styling is the problem (as inspecting DOM revealed that links are correctly generated).
Overriding the ``dt`` tag compromises the behavior of the ``a`` tag.

Reply:
hats off !